src/bufr/DataObject/ArrayDataObject.cpp

```diff
@@ -51,7 +51,7 @@ namespace Ingester
         params.chunk = true;
         params.chunks = chunks;
         params.compressWithGZIP(compressionLevel);
-        params.setFillValue<FloatType>(-999);
+        params.setFillValue<FloatType>(1.0e+11);
```

Comment:
Steve:  Ron and I had a discussion over email about this and there is a BUFR standard using just 1.E+11 as the indicator for missing value.  So that's why this works properly.  Otherwise, the ioda files were being given the 1.E+11 values as data while that line of code was saying the _FillValue was -999 so that was doing nothing useful downstream since it wasn't set to the proper missing value.  This change was something Ron addressed months ago in his feature branch that is awaiting other changes to submit.  Effectively he gave me a green light to pop in the fix and I adjusted all the test data outputs to match - so we're good to go.

Reply:
Thanks for the explanation @gthompsnJCSDA. I think it sounds good - I was not aware of a BUFR standard for missing values. I have a feeling that not all BUFR files from all providers will follow that standard but hopefully those are few and far between. I think it makes sense to go with the changes you made and address any missing values issues (if they come up) in a subsequent PR.